scripts/style-dictionary-css.cjs

```diff
@@ -109,7 +109,7 @@ function getStyleDictionaryConfig(theme) {
 
 console.log("Building tokens...");
 
-["cds-light", "cds-dark", "cds-lofi", "program", "mwc"].map((theme) => {
+["cds-tokens"].map((theme) => {
```

Comment:
So for any existing projects, we'll continue to use the component level tokens (which will no longer get updated), and new projects will use the semantic token? 

Were you thinking of doing themed semantic tokens later?

Reply:
> So for any existing projects, we'll continue to use the component level tokens (which will no longer get updated), and new projects will use the semantic token?

Yes, that's the assumption for now. I don't know that we have (m)any projects in progress that couldn't be delivered with the component-level tokens, nor any that would need to be maintained long-term and would therefore benefit from the new tokens approach. TR is one exception, but enough of it is "off" of the CDS that it seems like the LOE would be similar to implement either the existing components or the retokenized ones.

> Were you thinking of doing themed semantic tokens later?

Most likely, though I don't have a very specific timeframe in mind. See [my other comment](https://github.com/compassion-global-experience/cds-tokens/pull/43#issuecomment-1989053622) for more thoughts.